content/podcast/episode-03.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Episode 3: Year One of Limeleaf"
+date = 2025-03-24
+authors = ["John Luther", "Blain Smith", "Erik Straub"]
+template = "episode.html"
+description = "We started Limeleaf on March 18, 2024. To celebrate our one-year anniversary, we'll take a look back at what we've accomplished and where we want to go in 2025 and beyond."
+
+[extra]
+feature_photo = ""
+feature_photo_alt = ""
+
+[extra.podcast]
+episode = "3"
+source = "/podcast/limeleafpod-e03.mp3"
+type = "audio/mpeg"
+size = 14132634
+duration = "28:34"
+
+[extra.podcast.itunes]
+type = "full"
++++
+
+We started Limeleaf on March 18, 2024. To celebrate our one-year anniversary, we'll take a look back at what we've accomplished and where we want to go in 2025 and beyond.
+
+<!-- more -->
+
+## Show Notes
+- [Tech Jobs for Good](https://techjobsforgood.com/)
+- [comp.coop](https://comp.coop)
+- [Farox](https://www.farox.coop)
+- [Camplight](https://www.camplight.net/)
+- [Patio](https://patio.coop)
```

Comment:
Rushdown, Rocket Science, CoLab, apply, Limecast for completeness?

Reply:
Good call.